### PR TITLE
Add a recipe for pyevtk

### PR DESCRIPTION
**Pull request by renefritze**

### CODE REVIEW

**Comment by somada141** [comment]
@renefritze what's with the CI explosion :D ?

**Comment by renefritze** [comment]
> @renefritze what's with the CI explosion :D ?

Actually, I thought this was supposed to be just one build, since I've made the recipe `noarch` since we're pure python. This is all the conda-forge setup though, nothing I've added :wink:

**Comment by renefritze** [comment]
Ignoring the Azure failure on windows, from my end this would be good to merge @conda-forge/help-python

**Comment by hoechenberger** [comment]
@conda-forge/staged-recipes This PR is ready to be reviewed :)

File: recipes/pyevtk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  requires:
+    - pytest >=3.3
+
+  imports:
+    - pyevtk
+    - pyevtk.hl
+
+about:
+  home: https://github.com/pyscience-projects/pyevtk
+  license: BSD 2-clause
```
Comment (beckermr) [inline recipes/pyevtk/meta.yaml]: You need a license file entry.

Comment (renefritze) [reply recipes/pyevtk/meta.yaml]: Done: https://github.com/conda-forge/staged-recipes/pull/7718/commits/3e03c1d2c1d409b78c74b5bc5613914b92871fdd

**Comment by renefritze** [comment]
Pytest is indeed not used, because I realized midway through the process that I cannot them because they're not installed. I'll remove pytest from the test section. Please note that pytest-runner is still needed because our setup.py provides a test command through that.

**Pull request merged**